ods_ci/tests/Resources/Files/accelerator/qe_create_ap_3.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: dashboard.opendatahub.io/v1
+kind: AcceleratorProfile
+metadata:
+  name: qecreateap3
+  namespace: redhat-ods-applications
+spec:
+  description: description example
+  displayName: qe_create_ap_3
```

Comment:
Can you change values with a meaningful name here as well:
- name
- description 
- displayName

Reply:
Hi Fede, the description field is unclear.
The display name also. What is "ap_3" ?
Also, we cannot use underscore in K8s resources values, please use `-` instead.
See https://kubernetes.io/docs/concepts/overview/working-with-objects/names/

Reply:
Also, about the "description" field, as it is not mandatory, maybe it's a good idea to just leave it in the first case (where the user creates an instance manually using the UI).
WDYT @manosnoam ?

Reply:
I would make the description and displayname more relevant to the test used with that yaml.
For example, you can set displayName = "Testing accelerator with display name contains special characters". As well as the description - it should be relevant for the actual test. 

A good option would be to use `${TEST NAME}`and `${TEST DOCUMENTATION}` ([builtin variables in RF](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#automatic-variables)) as the description field value.
You can do it by replacing the value with [Jinja template mechanism](https://codeburst.io/jinja-2-explained-in-5-minutes-88548486834e) which we already support with our [openshift robot library](https://github.com/red-hat-data-services/robotframework-openshiftcli), for example:
`Oc Apply  AcceleratorProfile  qe_accelerator_profile_3.yaml  template_data={description: '${TEST DOCUMENTATION}', displayName: '${TEST NAME}'}`

Reply:
Also, I'm using DisplayName as a useful variable to manage all tests with only adding a number at the end:
`${ACC_DISPLAY_NAME}=    qe_accelerator_profile_X`
I'm doing it so then I can filter by that value easily